docs/testing-guidelines/getting-code-coverage.md

```diff
@@ -46,6 +46,23 @@ PS> Install-OpenCover -TargetDirectory $env:TEMP -Force
 PS> Invoke-OpenCover -OutputLog coverage.xml -OpenCoverPath $env:TEMP\OpenCover
 ```
 
+### Some notes if you are encountering issues
+
+`> Set-Location "C:\Path\to\powershell\build\dir"`
+
+Make sure there are no spaces in the path, since they are not replaced with escape characters.
+
+`> Install-OpenCover -TargetDirectory $env:TEMP -Force`
+
+If this throws an error: 
+
+`The request was aborted: Could not create SSL/TLS secure channel.` 
+
+try the following:
+
+`PS> [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12` 
```

Comment:
This should just be changed in the script I think. Windows PowerShell running on some .NET versions defaults to an old TLS library.

An example is:

https://github.com/PowerShell/PowerShell/blob/3f52adb0aeaf58189d85d386ab502e08f92ed41b/build.psm1#L1738

There are other places where it's done in a `try`/set -> `finally`/restore pattern.